.dockerignore

```diff
@@ -5,7 +5,9 @@ dev/dist
 dev/release
 python
 **/docs
-**/target
+target/
 **/CHANGELOG.md
 **/tests
 **/data
+!target/release/ballista-scheduler
+!target/release/ballista-executor
```

Comment:
Can't exclude the executables if we want them to go into the docker image.

Reply:
Is this change still needed now that the executables are build in Docker?

Reply:
Never mind ... after reviewing  the rest of the PR I understand why we have this